packages/css/src/components/button.css

```diff
@@ -0,0 +1,158 @@
+.button-base {
+  display: inline-block;
+  border-style: solid;
+  line-height: 1;
+  cursor: pointer;
+  appearance: none;
+}
+
+.button-base:focus {
+  outline: none;
+}
+
+.button-base:not([disabled]):active {
+  box-shadow: var(--button-shadow-active);
+}
+
+.button-base[disabled] {
+  cursor: not-allowed;
+}
+
+.button-base.medium {
+  font: var(--button-medium-font);
+}
+
+.button-base.large {
+  font: var(--button-large-font);
+}
+
+.button-master {
```

Comment:
People are moving away from the term `master` for various reasons, perhaps use `main` or just `button`